### PR TITLE
Add email validation success page

### PR DESCRIPTION
**Pull request by susiecoleman**
completeregistration

Implementing the page confirming the user has successfully registered.

### CODE REVIEW

File: public/register-confirmation-page.hbs [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{{# partial "content" }}
+
+  <div class="register-confirm">
+    <h1 class="register-confirm__heading">Now check your email</h1>
```
Comment (jamespamplin) [inline public/register-confirmation-page.hbs]: Can we please move the text into `messages.en`

**Comment by markjamesbutler** [comment]
@susiecoleman Could you please add a short description.  This helps others to understand the feature this PR is implementing.

File: public/register-confirmation-page.hbs (hunk 1 above)
Comment (susiecoleman) [inline public/register-confirmation-page.hbs]: Fixed

File: app/com/gu/identity/frontend/models/Text.scala [hunk 2]
```diff
@@ -66,4 +66,18 @@ object Text {
       )
     }
   }
+
```
Comment (susiecoleman) [inline app/com/gu/identity/frontend/models/Text.scala]: This object needs to be moved out to it's own file to match the structure of the text in #63

Comment (jamespamplin) [inline app/com/gu/identity/frontend/models/Text.scala]: Up to you whether you do this in this PR, or we can address it as tech debt. I already have a card for messages API tech debt.

File: public/register-confirmation-page.hbs [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{{# partial "content" }}
```
Comment (jamespamplin) [inline public/register-confirmation-page.hbs]: `pageTitle` partial needs to be set too, see: https://github.com/guardian/identity-frontend/blob/master/public/register-page.hbs#L1

Comment (susiecoleman) [inline public/register-confirmation-page.hbs]: Added

**Comment by jamespamplin** [comment]
:+1: great work!

**Pull request merged**